code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -484,18 +496,33 @@
 	matter = list(MATERIAL_STEEL = 3, MATERIAL_PLASTEEL = 2)
 	rarity_value = 25
 
+/obj/item/ammo_casing/grenade/frag/phosphorus
+	name = "SA WFR \"Micic\""
+	desc = "A military-grade White phosphorus grenade shell, designed to be fired from grenade launchers. The abbreviation \"SA\" marks its producer, but the lack of detatil makes it ubiquitous."
+	icon_state = "warcrime"
```

Comment:
No.

Reply:
Also, WFR?

Reply:
> No.

Yes

Reply:
> Also, WFR?

White phosphorus  round 
it should have been WPR but that was a typo